javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,83 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+//Displaying products in productsUl
+function renderProducts(products) {
+    let productLi, productName, productPrice, productRating;
+    products.forEach(product => {
+        productLi = document.createElement('li');
+        productName = document.createElement('h2');
+        productPrice = document.createElement('p');
+        productRating = document.createElement('p');
+
+        productName.innerHTML = product.name;
+        productPrice.innerHTML = `Price: ${product.price}`;
+        productRating.innerHTML = `Rating: ${product.rating}`;
+
```

Comment:
Get rid of extra blank lines. Less lines means less bugs.

Reply:
removed the extra lines from the code